Makefile

```diff
@@ -1,33 +1,31 @@
-BINDIR?=/usr/sbin
-MANDIR?=/usr/share/man
+BINDIR=/usr/sbin
+MANDIR=/usr/share/man
 
-.PHONY: install uninstall clean test
+.PHONY: release build install uninstall clean doc reindent
 
-all: main.native main.1
+release:
+	jbuilder build @install
 
-main.native: setup.data version.ml main.ml
-	ocaml setup.ml -build
+build:
+	jbuilder build @install --dev
 
-setup.data: _oasis
-	ocaml setup.ml -configure
+install:
+	install -m 0755 _build/install/default/bin/xapi-storage-script $(BINDIR)
+	install -m 0644 _build/install/default/man/man1/xapi-storage-script.1 $(MANDIR)/man1
```

Comment:
I'm not very sure this line is correct, because the spec file installs this under `man8`; is `xapi-storage-scirpt` a user or a system management utility?

Reply:
⤴️ ping @jonludlam or @robhoes

Reply:
@jonludlam says it belongs under `man8`. Also, a useful piece of info found at https://fedoraproject.org/wiki/Packaging:Guidelines#Manpages:

> As man pages are the traditional method of getting help on a Unix system, packages should contain them for all executables. If some man pages are absent, packagers should work with upstream to add them. It is also occasionally possible to find pages created by other distributions, or to use the output of the help2man program; those are often useful as a starting point. When installing man pages, note that they should be installed uncompressed as the build system will compress them as needed. The compression method may change, so it is important to reference the pages in the %files section with a pattern that takes this into account:
```
%files
%{_mandir}/man1/foo.1*
```